app/javascript/components/Home/Service/WorthPhotoContainer.1.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from "react";
+import { FadeInLeft } from "../../Constants/Constants";
+import {
+  postFetch,
+  deleteFetch,
+  getFetch
+} from "../../Constants/FetchComponent";
+import { DeleteButton, AddButton } from "../../Constants/Buttons";
+
+class WorthPhotoContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      url: "worth_photos",
+      photoData: [],
+      photo: "",
+      refreshKey: false
+    };
+  }
+
+  toggleRefreshKey = () => this.setState({ refreshKey: true });
+
+  onChange = e => this.setState({ [e.target.name]: e.target.value });
+
+  onSubmit = event => {
+    event.preventDefault();
+    const url = `/api/v1/${this.state.url}`;
+    const token = document.querySelector('meta[name="csrf-token"]').content;
+    const { photo } = this.state;
+    const body = { photo };
+
+    postFetch(url, token, body)
+      .then(this.toggleRefreshKey)
+      .catch(error => console.log("error message =>", error.message));
+  };
+
+  deleteEvent = id => {
+    const url = `/api/v1/${this.state.url}/${id}`;
+    const token = document.querySelector('meta[name="csrf-token"]').content;
+
+    deleteFetch(url, token)
+      .then(this.toggleRefreshKey)
+      .catch(error => console.log("error message =>", error.message));
+  };
+
+  componentDidMount() {
+    getFetch(this.state.url)
```

Comment:
It doesn't seem like `this.state.url` is changing at all. Instead of holding the value in state, you could try saving it as `const url = "urlhere.com"` and then just calling `url` instead of `this.state.url`

Reply:
totally true. the url was in the fetch itself before this. good call.

Reply:
did you mean a const in the did mount? I kept it up at state so it'd be more dynamic, changing that would change all the other fetches using it if i had to make an update to path.